File: arrays01.java
//arrays01 actual code for creating an Array to hold DVD's.
DVD[] dvdcollection = new DVD[15];

// A simple definition for a DVD.
public class DVD{

  public String name;
  public int releaseYear;
  public String director;
  

  public DVD(String name, int releaseYear, String director){
    this.name = name;
    this.releaseYear = releaseYear;
    this.director = director;

  }

  public String toString(){
    return this.name + ", directed by" + this.director + ", relased in" + this.releaseYear;
  }
  
}

File: arrays01.py
# Define a class for DVD
class DVD:
  def __init__(self, name, release_year, director):
    self.name = name
    self.release_year = release_year
    self.director = director

  
  def __str__(self):
    return f"{self.name}, directed by {self.director}, released in {self.release_year}"

# Create a list to hold up to 15 DVDs
dvd_collection = [None] * 15

# Example of adding a DVD to the collection
dvd_collection[0] = DVD("Inception", 2010, "Christopher Nolan")

# Print the details of the first DVD in the collection
print(dvd_collection)

print(len(dvd_collection))